tests/steamship_tests/utils/test_key_value_store.py

```diff
@@ -0,0 +1,237 @@
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from steamship_tests.utils.client import get_steamship_client
+from steamship_tests.utils.random import random_name
+
+from steamship import Block, File, Space, Steamship, Tag
+
+
+class KeyValueStore:
+    """A simple key value store implemented in Steamship.
+
+    Each entry is stored as an empty file whose "Key" has:
+      * Kind = KeyValueStore
+      * Name = the key of the (kv) pair
+      * Value = a dict set to the value
+
+    This class is stored inside the `tests` folder because:
+      1) It has proven useful enough in package development that we wish to make sure it's tested, and
+      2) We wish to add KV-functionality to Spaces in general, but
+      3) We're not eager to directly add this as a utility class in the codebase.
+
+    For the time being; consider this a copy-paste reference implementation along with tests.
+    """
+
+    client: Steamship
+    namespace: str
+
+    def __init__(self, client: Steamship, namespace: str = "KeyValueStore"):
```

Comment:
Would add docs to explain namespace != workspace, instead it uniquely identifies the key-value store you will use. You can have more than one KV store in a single workspace.

Reply:
Good call -- renamed the variable and added a note about having multiple.